terraform/grafana-folders.tf

```diff
@@ -1,7 +1,11 @@
+resource "grafana_folder" "aegis" {
+  title = "Aegis"
+}
+
 resource "grafana_folder" "oracle_relayers" {
   title = "Oracle Relayers"
 }
 
-resource "grafana_folder" "aegis" {
-  title = "Aegis"
+data "grafana_folder" "reserve" {
```

Comment:
what's the difference here between `resource` and `data` for the folders?

Reply:
there was a pre-existing `reserve` folder in grafana already that was manually created sometime, `data` just means "use this resource that's already there" (whereas `resource` means "create this new resource")